uint32 id
float32 acceleration
float32 steeringAngle